gcp_deepvariant_runner.py

```diff
@@ -256,25 +314,34 @@ def _is_valid_gcs_path(gcs_path):
   Args:
     gcs_path: (str) a path to directory or an obj on GCS.
   """
-  return urlparse.urlparse(gcs_path).scheme == 'gs'
+  return (urlparse.urlparse(gcs_path).scheme == 'gs' and
+          urlparse.urlparse(gcs_path).netloc)
```

Comment:
This change (and other related parts of that commit) should either be a separate PR or you should squash your commits so the history is meaningful after.